### DISCUSSION TITLE
Works local but not on ado

### DISCUSSION THREAD

**Discussion by ericfabiszewski** [artilleryio/artillery#1352] [Artillery]
I am very new with Artillery but I wrote a test that i was able to run locally and when i tried to run it through a pipeline on azure, is there any known issues for that?  Or am i just doing it wrong

config:
  target: "endpoint url"
  phases:
    - duration: 60
      arrivalCount: 30

scenarios:
  - name: "GET"
    flow:
       - log: "Getting token"
       - post:
          url: "token url"
          headers:
            Content-Type: "application/x-www-form-urlencoded"
          form:
           username: 'xxx@xxx.com'
           password: 'password123'
           grant_type: 'password'
           client_id: 'xxxxxxx'
           client_secret: 'xxxxxxxxxxx'
           scope: 'xxxxxx'
           subscriberId: 'xxxxx'
           isStaff: 'true'
          capture:
           json: "$.access_token"
           as: access_token
          expect:
            - statusCode: 200
       - log: "Token : {{access_token}}"
       - get:
          url: "***another api***"
          headers:
            Authorization: "Bearer {{access_token}}"
            locationId: 
       - think: 5

Figured this out

**Comment by hassy** [comment]
hi @ericfabiszewski 👋 what was the issue?

**Reply by ericfabiszewski** [reply]
had to use one of our  custom agents to be connected to the network before i attempted to login
